main.tf

```diff
@@ -136,14 +146,24 @@ resource "azurerm_windows_virtual_machine" "windows" {
     }
   }
 
+  boot_diagnostics {
```

Comment:
duplicate entry from above.

Reply:
needed for both linux and windows vm resource